WolvenKit.App/ViewModels/Tools/ImportExportViewModel.cs

```diff
@@ -999,6 +998,13 @@ private void ResetSettings()
         [RelayCommand]
         private void ImportSettings()
         {
+            // Don't import if there is no active project.
+            if (_projectManager.ActiveProject == null
+                || string.IsNullOrWhiteSpace(_projectManager.ActiveProject.ProjectDirectory))
+            {
+                return;
```

Comment:
Maybe we can log an error here or a notification? Or add a CanExecute to the command?

Reply:
![image](https://user-images.githubusercontent.com/1414138/193842706-06ba30d5-61c7-4aad-9502-baab80a19b6a.png)

Reply:
We are getting disabled which is ideal, but its not... refreshing once there is an ActiveProject. Looking for solution.

Reply:
Add `ImportSettingsCommand.NotifyCanExecuteChanged();` to `this.WhenAnyValue(x => x._projectManager.ActiveProject).Subscribe`

Reply:
Nice I will check it out, can be done on `null` member though right (since it stats of null)?